build.gradle

```diff
@@ -1,11 +1,13 @@
 buildscript {
     repositories {
         mavenCentral()
+        maven { url 'https://jitpack.io' }
     }
 
     dependencies {
         // https://github.com/melix/japicmp-gradle-plugin/issues/36
         classpath 'com.google.guava:guava:30.1.1-jre'
+        classpath 'com.github.kiview:captain-hook:76a1c11a16'
```

Comment:
@kiview do you remember why we had to fork it?

Reply:
to make it work with windows I guess https://github.com/kiview/captain-hook/commits/master

Reply:
Exactly, PR was never merged in upstream, since upstream seems to be dead 😞